CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 3.1 (Next)
 
-* Your contribution here.
+* Inserted missing space between `${searchExistingType}` and `issues`.
```

Comment:
Make it look like the other entries below with the GH issue link? 

I’ll add danger-changelog to get this automated …